bin/Debugger.Sample/Debugger.Sample.cpp

```diff
@@ -11,16 +11,18 @@ class CommandLineArguments
 public:
     bool breakOnNextLine;
     bool enableDebugging;
-    int port;
     bool help;
+    bool disableConsoleRedirect;
+    int port;
 
     std::vector<std::wstring> scriptArgs;
 
     CommandLineArguments()
         : breakOnNextLine(false)
         , enableDebugging(false)
-        , port(9229)
         , help(false)
+        , disableConsoleRedirect(false)
```

Comment:
Instead of double-negative of disableConsoleRedirect of false meaning to do redirect, rename the variable to something positive such as "enableConsoleRedirect" with default being true and have the switch make it false.